### PR TITLE
Create FunctionSignatures.md

### PR DESCRIPTION
**Pull request by cupOJoseph**
adds a new docs md with Function Signatures list

### CODE REVIEW

**Comment by cupOJoseph** [comment]
h/t @BCLeFevre

File: docs/FunctionSignatures.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Function Signatures
+
+0xfb0f3ee1 = fulfillBasicOrder\
+0xb3a34c4c = fulfillOrder\
+0xe7acab24 = fulfillAdvancedOrder\
+0xa8174404 = matchOrders\
+0x55944a42 = matchAdvancedOrders\
+0xed98a574 = fulfillAvailableOrders\
+0x87201b41 = fulfillAvailableAdvancedOrders\
```
Comment (ryanio) [inline docs/FunctionSignatures.md:9]:
```suggestion
0x87201b41 = fulfillAvailableAdvancedOrders
```

File: docs/FunctionSignatures.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+0xfd9f1e10 = cancel\
+0x88147732 = validate\
+0x5b34b966 = incrementCounter\
```
Comment (ryanio) [inline docs/FunctionSignatures.md:13]:
```suggestion
0x5b34b966 = incrementCounter
```

**Review by ryanio — APPROVED** [review]
lgtm!

**Pull request merged**